.gitignore

```diff
@@ -43,6 +43,7 @@ deps/
 teamsyncd/teamsyncd
 fpmsyncd/fpmsyncd
 intfsyncd/intfsyncd
+cfgagent/intfconfd
```

Comment:
can you rename to intfmgr? 

The role of confd, orchagent, syncd are blurred, it is hard to have a clear cuts of different roles they are playing, so mgr can include all functions, which manages both asic and linux setup. therefore I prefer mgr name

Reply:
Ok, will rename them to intfmgr and vlanmgr.  For switch table level configuration,  besides switchmgr process itself,  the configuration may be used by multiple function modules like VLAN.

I plan to keep the class name of VlanConf, IntfConf and SwitchConf,  let me know if you have other suggestions.